Lombiq.Tests.UI/BasicOrchardFeaturesTesting/WorkflowsFeatureTestingUITestContextExtensions.cs

```diff
@@ -69,6 +70,15 @@ public static class WorkflowsFeatureTestingUITestContextExtensions
 
                 // Checking if the workflow run was logged.
                 await context.GoToAdminRelativeUrlAsync(workflowsPath);
+
+                // If we are testing an app with already existing workflows, our "Test workflow" can end up on a
+                // different page, rather than on the first.
+                await context.ClickAndFillInWithRetriesAsync(By.Id("Options_Search"), testWorkflowName);
+
+                // Normally we would trigger filtering by pressing the "Enter" key. The filter submit button is hidden,
+                // so we have to use JS to click on it.
+                context.ExecuteScript("document.querySelector('#submitFilter').click();");
```

Comment:
Don't you want to turn this into an extension method? Also JS has a dedicated method for finding by ID:
```suggestion
                context.ExecuteScript("document.getElementById('submitFilter').click();");
```